hier_motif_feature_generator.py

```diff
@@ -77,7 +77,8 @@ def get_possible_motifs_to_targets(self, mask_shape):
         # We cannot have a motif mutate to the same center nucleotide
         theta_mask = np.ones(mask_shape, dtype=bool)
         if mask_shape[1] > 1:
-            for i, (motif, mutating_pos) in enumerate(self.feature_info_list):
+            for i, (motif, distance_to_start_of_motif) in enumerate(self.feature_info_list):
+                mutating_pos = -distance_to_start_of_motif
                 mutating_nucleotide = motif[mutating_pos]
```

Comment:
weird things are going to happen when this is positive vs negative. have you thought through those cases? do you want both the positive and negative values for `distance_to_start_of_motif`?